framework_tvb/tvb/interfaces/web/templates/jinja2/user/profile.html

```diff
@@ -32,7 +32,7 @@ <h3>User Profile</h3>
 
     <section class="profile">
         <h2><mark>User</mark> details</h2>
-		{% if showOnlineHelp %}
+		{% if showOnlineHelp and keycloakConfigWebFile is not defined %}
```

Comment:
I don't understand what is the change doing here. Pls explain

Reply:
This change is hiding the online help question mark near the User Details in the user profile page when keycloak is enabled.

Reply:
... because the help on this section mentions TVB users and that is no longer relevant in this deployment. Long term we should update the help to contain info about the keycloak login. But that is not something which we should fix now.